drizzlepac/haputils/analyze.py

```diff
@@ -104,6 +142,9 @@ def analyze_data(input_file_list, log_level=logutil.logging.NOTSET):
         The desired level of verboseness in the log statements displayed on the screen and written to the .log file.
         Default value is 20, or 'info'.
 
+    type : string
```

Comment:
It might be helpful for us later (if no one else) to include the list of supported possible values for this string here.

Reply:
Fixed in commit 111c243.